fms/models/llama.py

```diff
@@ -442,65 +445,6 @@ def _hf_sd_to_fms_sd(hf_sd: Mapping) -> Mapping:
 serialization.register_adapter("llama", "hf", _hf_sd_to_fms_sd)
 
 
-def load_fms_llama(model_path: str, group=None, **kwargs):
```

Comment:
!!

I guess I'm glad to see this go but it also surprises me haha

Reply:
I think it is time... lol